lib/rouge/lexers/prolog.rb

```diff
@@ -13,6 +13,7 @@ class Prolog < RegexLexer
       state :basic do
         rule /\s+/, Text
         rule /^#.*/, Comment::Single
+        rule /%.*/, Comment::Single
```

Comment:
Is a comment starting with `#` required to be at the beginning whereas `%` can be a trailing comment? I'm trying to understand if there should be a different between the two rules here as written.

Reply:
Will you also please add a comment example with `%` in `spec/visual/prolog`?

Reply:
As far as I know, `#` isn't actually a comment character at all in Prolog -- I suppose that's just there to make "shebangs" (`#!`) be highlighted properly?

`%` can indeed be either at the beginning of a line or trailing.